core/variables.js

```diff
@@ -75,6 +76,22 @@ Blockly.Variables.allVariables = function(root) {
 };
 
 /**
+ * Find all variables that the user has created through the workspace or
+ * toolbox.  For use by generators.
+ * @param {!Blockly.Workspace} root The workspace to inspect.
+ * @return {!Array.<string>} Array of variable names.
+ */
+Blockly.Variables.allVariables = function(root) {
+  if (root.getDescendants) {
```

Comment:
Because it's what was used before (old line 46/new line 47).  Want me to change both and then commit?